File: SEM2/College_Codes/Functions/Exercise4.c
#include<stdio.h>
#include<math.h>
int convert(int binary)
{
    int temp=binary,remainder=0,decimal=0,i=0;
    while (temp!=0)
    {
        remainder=temp%10;
        temp/=10;
        decimal=decimal+remainder*pow(2,i);
        i++;
    }
    printf("Equivalent decimal number is %d.",decimal);
}
int main()
{
    int binary;
    printf("Enter binary number: ");
    scanf("%d",&binary);
    convert(binary);
    return 0;
}

File: SEM3/PYTHON/College_Codes/Control Statements/exercise4.py
flag=1

while(flag==1):
    no = input("Enter the number: ")
    length = len(no)

    summ=0
    temp = int(no)
    while(temp!=0):
        remainder=temp%10
        summ+=pow(remainder,length)
        temp=temp//10

    if(summ==int(no)):
    	print(no + " is an armstrong number.")
    else:
    	print(no + " is not an armstrong number.")

    intake=input("Do you wish to continue? (Press y to continue or any key to exit.): ")

    if(intake=="Y" or intake=="y"):
        flag=1
    else:
        flag=0
        break